.gitignore

```diff
@@ -26,3 +26,5 @@ install-sh
 missing
 pluglib-bindings.ami
 *.pot
+coverage/
```

Comment:
is this one needed? I think only one should be enough and also please use `/coverage` so it match only top level.

Reply:
I just copied this values from the [`.gitignore` available in master](https://github.com/yast/yast-firstboot/blob/master/.gitignore) :)

But, for sure, we can improve it :+1: